iothub/device/src/Transport/Http/HttpClientHelper.cs

```diff
@@ -300,99 +147,20 @@ private static void InsertEtag(HttpRequestMessage requestMessage, IETagHolder en
             return result;
         }
 
-        private Task PostAsyncHelper<T1>(
+        private async Task PostAsyncHelper<T1>(
```

Comment:
If ```PostAsync``` is a one liner that just calls this helper, why not delete the helper and move its contents into the ```PostAsync``` method?